openwisp_controller/connection/tasks.py

```diff
@@ -8,12 +8,14 @@
 from django.utils.translation import gettext_lazy as _
 from swapper import load_model
 
+from openwisp_utils.tasks import OpenwispCeleryTask
+
 from . import settings as app_settings
 
 logger = logging.getLogger(__name__)
 
 
-@shared_task
+@shared_task(base=OpenwispCeleryTask)
```

Comment:
this is a network operation, not a background task and hence `OpenwispCeleryTask` does not apply here